ui-frontend/packages/catalog-ui-search/src/main/webapp/js/model/User.js

```diff
@@ -499,7 +499,7 @@ User.Response = Backbone.AssociatedModel.extend({
         return this.canWrite(thing.get('metacard').get('properties'))
       case 'query-result.collection':
       default:
-        if (this.some !== undefined) {
+        if (thing.some !== undefined) {
           !thing.some(subthing => {
```

Comment:
📝  I think the preferred solution here would be to exit out of this as soon as one result isn't editable. 
It would still be slow in the case all are editable, but it should lessen the load time for mixed results or when all aren't editable.